<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmPurchasingAuto</class>
 <widget class="QDialog" name="ConfirmPurchasingAuto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirming purchase</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/check.png</normaloff>:/img/check.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
                    background:qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                          stop:0.03 rgba(184,193,201,1), 
                          stop:0.27 rgba(107,150,191,1), 
                          stop:0.92 rgba(245,39,39,1));
}
         
   </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="confirmPurchasingPb">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Heavy</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">

QPushButton{
border: solid 0px #ffffff;
background-color: rgb(83, 144, 190);;
border-radius:18px;
margin-left:35px;
margin-right:35px;
color:rgb(255, 255, 255);
}

QPushButton:hover{
border: 1.5px inset rgb(1, 1, 1);
background-color: rgb(59, 104, 136);
border-radius:18px;
}</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color:rgb(245, 245, 245);
	border:0px solid;
	border-radius:20px;
}

QLineEdit{
	background-color:rgb(110, 151, 190);
	font: 350 13pt &quot;Bahnschrift SemiLight SemiConde&quot;;
	color:rgb(1, 1, 1);
	border:0px;
padding-left:15px;
	border-radius:15px;
}

QLineEdit:focus{
	background-color:rgb(83, 107, 135);
	font: 350 13pt &quot;Bahnschrift SemiLight SemiConde&quot;;
	color:rgb(1, 1, 1);
	border:2px solid rgb(0, 0, 0);
	border-radius:15px;
}

</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="1">
       <widget class="QLineEdit" name="autoYearLn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Year:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="autoPriceLn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Price of auto($):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="autoNumberLn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Number of auto:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="autoBrandLn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Brand:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Franklin Gothic Heavy</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color:rgba(245,39,39,1);
border:0px;
margin-top:5px;
};</string>
        </property>
        <property name="text">
         <string>Detailed info:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="autoModelLn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="1">
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
